packages/pypy3.rb

```diff
@@ -0,0 +1,46 @@
+require 'package'
+
+class Pypy3 < Package
+  description 'pypy is fast, drop-in replacement of cpython that can execute code over 4 times faster.'
+  homepage 'https://www.pypy.org/'
+  version '7.3.3'
+  compatibility 'x86_64, i686, aarch64'
+  
+  case ARCH
+  when 'x86_64'
+    source_url 'https://downloads.python.org/pypy/pypy3.7-v7.3.3-linux64.tar.bz2'
+    source_sha256 '37e2804c4661c86c857d709d28c7de716b000d31e89766599fdf5a98928b7096'
+  when 'i686'
+    source_url 'https://downloads.python.org/pypy/pypy3.7-v7.3.3-linux32.tar.bz2'
+    source_sha256 '7d81b8e9fcd07c067cfe2f519ab770ec62928ee8787f952cadf2d2786246efc8'
+  when 'aarch64'
+    source_url 'https://downloads.python.org/pypy/pypy3.7-v7.3.3-aarch64.tar.bz2'
```

Comment:
This won't work on aarch64 since it operates under a 32-bit userland.

Reply:
okay i'll remove it